types/rsvp/tsconfig.json

```diff
@@ -16,6 +16,7 @@
         ],
         "types": [],
         "noEmit": true,
+        "exactOptionalPropertyTypes": true,
```

Comment:
Why did you also update the config for this package? I can see how the others relate to ember, but not sure about this one.

Reply:
This is a fair question! It turns out that the primary consumer of RSVP *is* Ember (most other folks are at this point just using native Promises *or* something like Bluebird), and this is upstream of massive amounts of Ember APIs as a result.

Reply:
So I should leave it in, right, @chriskrycho?